<ui version="4.0" >
 <class>MainTxtWindow</class>
 <widget class="QWidget" name="MainTxtWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtposX" >
        <property name="inputMask" >
         <string/>
        </property>
        <property name="text" >
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtposY" >
        <property name="text" >
         <string>Y</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="txtbuttonSetPos" >
      <property name="text" >
       <string>Set pos</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="txtsizeLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtsizeX" >
          <property name="inputMask" >
           <string/>
          </property>
          <property name="text" >
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtsizeY" >
          <property name="text" >
           <string>Y</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="txtbuttonSetSize" >
        <property name="text" >
         <string>Set tama√±o</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="txtbuttonSetVelocity" >
          <property name="text" >
           <string>Set Velocidad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txttextVelocity" />
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="txtbuttonSetFont" >
      <property name="text" >
       <string>Set Fuente</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="txtbuttonSetBackColor" >
      <property name="text" >
       <string>Set backcolor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="txtbuttonSetFontColor" >
      <property name="text" >
       <string>Set fontcolor</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="txtbuttonStart" >
        <property name="text" >
         <string>Comenzar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="txtbuttonStop" >
        <property name="text" >
         <string>Frenar</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="txtbuttonHideWindow" >
          <property name="text" >
           <string>Esconder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="txtbuttonShowWindow" >
          <property name="text" >
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>txtbuttonSetPos</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>txtbuttonSetFont</zorder>
   <zorder>txtbuttonSetBackColor</zorder>
   <zorder>txtbuttonSetFontColor</zorder>
   <zorder>verticalLayoutWidget_5</zorder>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Soltar archivos aca =====></string>
      </property>
      <property name="scaledContents" >
       <bool>true</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txttextFileReciber" >
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="undoRedoEnabled" >
       <bool>true</bool>
      </property>
      <property name="lineWrapMode" >
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="acceptRichText" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>txttextFileReciber</zorder>
   <zorder>label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>verticalLayoutWidget_4</zorder>
   <zorder>verticalLayoutWidget_6</zorder>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="font" >
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Cola de mensajes</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="txtbuttonDeleteSms" >
      <property name="text" >
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>txtbuttonDeleteSms</zorder>
   <zorder>label_3</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>txtbuttonSetFont</zorder>
   <zorder>txtbuttonSetFontColor</zorder>
   <zorder>txtbuttonSetBackColor</zorder>
   <zorder>verticalLayoutWidget_5</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>change_size(int,int)</slot>
 </slots>
</ui>
